frontend/translations/en.po

```diff
@@ -68,6 +68,12 @@ msgstr "Great to see you again!"
 msgid "auth.login-with-github-submit"
 msgstr "Login with Github"
 
+msgid "auth.login-with-google-submit"
+msgstr "Login with Google"
+
+msgid "auth.login-with-oidc-submit"
+msgstr "Login with OpenID (SSO)"
+
```

Comment:
We should write always spanish translation at least. It's makes not much sense to add it later.